package.json

```diff
@@ -56,6 +56,7 @@
     "babel-jest": "^27.4.6",
     "chromatic": "^6.4.3",
     "css-loader": "^5.2.7",
+    "dompurify": "^2.3.6",
```

Comment:
We are adding a sanitising library? We had discussed not doing this within these components in other tickets. Was there a change?

Reply:
@matthewroach You're suggesting then that sanitising would then take place on the block level?

Reply:
Currently we don't do any sanitising, and that was one of the reasons we've yet to decided where it would be done

Reply:
gotcha no problem